css.css

```diff
@@ -0,0 +1,128 @@
+body {
+  background: rgb(188,86,188);
+background: radial-gradient(circle, rgba(188,86,188,1) 0%, rgba(150,182,255,1) 100%);
+}
+.container {
+  width: 1200px;
+  height: 100%;
+  margin: auto;
+  padding-top: 30px;
+}
+
+/* Search */
+.search {
+
+}
+
+.search h2 {
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+
+
+/* Tests */
+#TestsAndResult {
+
+}
+
+.wrapper {
+  display: flex;
+}
+
+/* Arguments */
+#tests {
+  margin-right: 40px;
+}
+
+#testsList {
```

Comment:
This considered a bad style to add css to the ids.
It is better to use classes. Even if class duplicates id name.